src/Button/Button.tsx

```diff
@@ -47,35 +47,35 @@ function getButtonProps(
   }
 }
 
-const _Button: React.FC<ButtonProps> = ({
-  className,
-  color = "primary",
-  error,
-  variant = "tertiary",
-  ...props
-}) => {
-  const classes = useStyles();
+const _Button: React.FC<ButtonProps> = React.forwardRef(
+  (
+    { className, color = "primary", error, variant = "tertiary", ...props },
+    ref
+  ) => {
+    const classes = useStyles();
 
-  return (
-    <MuiButton
-      className={clsx(className, {
-        [classes.primary]: variant === "primary" && error,
-        [classes.primaryDisabled]:
-          variant === "primary" && error && props.disabled,
+    return (
+      <MuiButton
+        ref={ref}
```

Comment:
Can you confirm it is actually working? I think this has to be `innerRef`
```suggestion
        innerRef={ref}
```

Reply:
Yes, it is working